core/pom.xml

```diff
@@ -127,7 +127,7 @@
 		<dependency>
 			<groupId>org.springframework.security.oauth</groupId>
 			<artifactId>spring-security-oauth2</artifactId>
-			<version>2.5.0.RELEASE</version>
+			<version>${spring.security.oauth2.version}</version>
```

Comment:
Can the version be defined in the parent pom?

Reply:
It is defined in parent pom.